eng/Versions.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <!-- This repo version -->
+    <!-- We need to modify the version here manually everytime the version changes from the upstream branch jbevain/cecil -->
+    <VersionPrefix>0.11.4.0</VersionPrefix>
+    <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
```

Comment:
nit: in other repos where we're just building a transport package we're using `alpha` as the PreReleaseVersionLabel